README.md

```diff
@@ -1,2 +1,53 @@
 # census-rm-fieldwork-adapter
-Adapter service to convert from JSON events to legacy XML format required by FWMT.
+Adapter service to convert from JSON events to the XML format required by Fieldwork Management Tool (FWMT).
```

Comment:
I does also have the responsibility of turning things like an invalid address into a "new" fieldwork case. Or indeed a receipt into a "cancel" fieldwork case - business logic worth mentioning.

Reply:
Ok, I've added that detail.